### ISSUE TITLE
NameError: name 'box' is not defined

### ISSUE THREAD

**Issue by hainm** [ParmEd/ParmEd#197]
Error, man
```
haichit@X ~/test $ python  test_parmed_pdb_box.py
[55.451, 42.736, 26.926, 90.0, 90.0, 90.0]
Traceback (most recent call last):
  File "test_parmed_pdb_box.py", line 5, in <module>
    print (pdb)
  File "/home/haichit/anaconda3/lib/python3.4/site-packages/parmed/structure.py", line 2830, in __str__
    return repr(self)
  File "/home/haichit/anaconda3/lib/python3.4/site-packages/parmed/structure.py", line 325, in __repr__
    if (abs(box[3]-90) > SMALL or abs(box[4]-90) > SMALL or
NameError: name 'box' is not defined
```

code
```python
import parmed as pmd

pdb = pmd.download_PDB("1L1H")
print (pdb.box)
print (pdb)
```

ParmEd version: just pull from master.

**Comment by swails** [comment]
Man I'm dumb.  And that's why I should use pyflakes before I commit.  See #198

**Comment by hainm** [comment]
so I guess I should use pyflakes too.

**Comment by hainm** [comment]
nvm, it pyflakes does not work with Cython code.

**Comment by rmcgibbo** [comment]
> And that's why I should use pyflakes before I commit.

Or put `pyflakes` in the travis.yml test section, if you're feeling masochistic.

**Comment by swails** [comment]
> Or put pyflakes in the travis.yml test section, if you're feeling masochistic.

My one complaint about pyflakes is that it really doesn't like you importing stuff into a package namespace.  It complains about *every* import I have in `__init__.py` files because I'm "not using them"... no kidding pyflakes.

**Comment by swails** [comment]
Fixed by #198
